### PR TITLE
REFACTOR: Create libcoveweb2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: cove_ofds/process.py [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+        super().__init__(supplied_data)
+        self.nodes_file_name = os.path.join(
+            self.supplied_data.data_dir(), "nodes.geo.json"
+        )
+        self.spans_file_name = os.path.join(
+            self.supplied_data.data_dir(), "spans.geo.json"
+        )
+
+    def process(self, process_data: dict) -> dict:
+        # TODO if original format this, don't bother
```
Comment (Lathrisk) [inline cove_ofds/process.py:187]: It's not clear to me what this comment means. Should it be?

**Review by Lathrisk — APPROVED** [review]
LGTM, I'm a little unsure of the use of the word "new" in the context of "NewGeoJSON" or "NewSpreadsheet" etc, but not a blocker.

File: cove_ofds/process.py (hunk 1 above)
Comment (ghost) [reply cove_ofds/process.py:187]: "if user uploaded GeoJSON, don't bother to convert to GeoJSON" - I'll sort later

**Comment by ghost** [comment]
Thanks - I'll ask about new - is that in URL/View contxets? can find something better

**Comment by Lathrisk** [comment]
Yeah, it felt a little ambiguous given all of the files are `new`. I guess it depends on context, but `supplied`, or `source`, etc?

**Pull request merged**